src/dev/DevFlagsBox.tsx

```diff
@@ -40,15 +41,15 @@ const Gate = ([flagName, flagSetting]: [string, string]) => {
 }
 
 export default function DevFlagsBox() {
-  const featureFlagsAtom = useAtomValue(featureFlagSettings)
-  const featureFlags = useMemo(() => Object.entries(featureFlagsAtom), [featureFlagsAtom])
+  const featureFlagsAtom = useAtomValue(featureFlagSettingsAtom)
+  const featureFlags = useMemo(() => Object.values(FeatureFlag), [])
```

Comment:
needed to replace this with the enum values instead of the atom values -- because atom is initialized empty